pkgs/applications/audio/vital/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, makeWrapper
+, unzip
+
+, alsa-lib
+, libjack2
+, curl
+, xorg
+, libGL
+, freetype
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vital";
+  version = "1.5.5";
+
+  unpackCommand = ''
+    mkdir -p root
+    unzip $curSrc root
+  '';
+
+  src = fetchurl {
+    url = "https://builds.vital.audio/VitalAudio/vital/${builtins.replaceStrings ["."] ["_"] finalAttrs.version}/VitalInstaller.zip";
+    hash = "sha256-aPPH6EXz16W0SoOt62407yIVA98A55ZPfVofEyolLRM=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    makeWrapper
+    unzip
+  ];
+
+  buildInputs = [
+    alsa-lib
+    stdenv.cc.cc.lib
+    libGL
+    xorg.libSM
+    xorg.libICE
+    xorg.libX11
+    freetype
+    libjack2
+  ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    # copy each output to its destination (individually)
+    mkdir -p $out/{bin,lib/{clap,vst,vst3}}
+    for f in bin/Vital lib/{clap/Vital.clap,vst/Vital.so,vst3/Vital.vst3}; do
+      cp -r $f $out/$f
+    done
+
+    wrapProgram $out/bin/Vital \
+      --prefix LD_LIBRARY_PATH : "${lib.makeLibraryPath [ curl libjack2 ]}"
+
+    runHook postInstall
+  '';
+
+  passthru.updateScript = ./updater.sh;
+
+  meta = with lib; {
+    description = "Spectral warping wavetable synth.";
+    homepage = "https://vital.audio/";
+    sourceProvenance = with sourceTypes; [ binaryNativeCode ];
+    license = licenses.unfree;
```

Comment:
I found the [github repository](https://github.com/mtytel/vital) for older releases and it says that the license of vital is a gpl3Plus.
Should we use the dual license notation here?

Reply:
Possibly. The source code release of Vital hasn't been updated in quite some time. (It is for version 1.0.7.) Hopefully it will receive updates soon, but until that happens, I would consider Vital to be proprietary software. Not to mention the assets that Vital comes with that are not allowed to be included in FOSS versions of the codebase, such as DISTRHO's Vitalium. What do you think? It could be worth including.

Reply:
You know, I think I will mark it as dual license. It's worth noting it at least.

Reply:
We can inform others that the source code may be available if necessary.